features/AuthController.feature

```diff
@@ -19,6 +19,32 @@ Feature: Auth Controller
       | valid | false      |
       | state | "_STRING_" |
 
+  @security @http
+  Scenario: Check if a token in the cookie is valid
+    Given I send a HTTP "POST" request with:
+      | controller | "auth"                                               |
+      | action     | "login"                                              |
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
+      | cookieAuth | true                                                 |
+    When I send a HTTP "POST" request with:
+      | controller | "auth"                                                      |
+      | action     | "checkToken"                                                |
+      | cookieAuth | true                                                        |
+      | headers    | { cookie: this.props.rawResponse.headers['set-cookie'][0] } |
+    Then I should receive a result matching:
+      | valid     | true         |
+      | kuid      | "test-admin" |
+      | expiresAt | "_NUMBER_"   |
+    When I send a HTTP "POST" request with:
+      | controller | "auth"                                                      |
+      | action     | "checkToken"                                                |
+      | cookieAuth | true                                                        |
+      | headers    | { cookie: 'authToken=wrongtoken' }                                              |
```

Comment:
Nitpicking: you can construct object "on the fly"
```suggestion
      | headers.cookie    | 'authToken=wrongtoken'                                              |
```

Reply:
Has I said in your previous comment  I tried, but this doesn't seem to work